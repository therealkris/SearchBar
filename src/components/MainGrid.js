// @flow
import * as React from "react";

import styles from "./styles.module.scss";

export default function MainGrid() {
  return (
    <div className={styles.mainGrid}>
      <div />
      <div />
      <div />
    </div>
  );
}
