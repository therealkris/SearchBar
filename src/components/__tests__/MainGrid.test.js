import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";

import MainGrid from "../MainGrid";

test("renders correctly", () => {
  const { container } = render(<MainGrid />);
  expect(container).toMatchSnapshot();
  expect(container).toContainHTML("<div></div>");
});
