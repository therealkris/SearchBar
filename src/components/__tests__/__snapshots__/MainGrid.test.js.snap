// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="mainGrid"
  >
    <div />
    <div />
    <div />
  </div>
</div>
`;
